src/main/scala/ch/epfl/bluebrain/nexus/sbt/nexus/MonitoringPlugin.scala

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.bluebrain.nexus.sbt.nexus
+
+import sbt.Keys.libraryDependencies
+import sbt._
+
+object MonitoringPlugin extends AutoPlugin {
+
+  override lazy val requires = empty
+
+  override lazy val trigger = allRequirements
+
+  trait Keys {
+    val kamonVersion = SettingKey[String]("kamon-version", "Kamon version")
+  }
+
+  object autoImport extends Keys
+  import autoImport._
+
+  override lazy val projectSettings = Seq(
+    kamonVersion := "1.0.0",
```

Comment:
this doesn't work with kamon as it versions its modules independently; for example, right now the jaeger module is at 1.0.1 while the prometheus one is at 1.0.0.

this needs independent setting keys for all used modules